src/services/sceneManager.ts

```
@@ -1764,16 +1696,70 @@ export class SceneManager {
+    material.dispose();
+  }
+
+  // Helper method to dispose a single mesh (geometry + material + textures)
+  private disposeMesh(mesh: THREE.Mesh): void {
+    if (mesh.geometry) {
+      mesh.geometry.dispose();
+    }
+    if (mesh.material) {
+      if (Array.isArray(mesh.material)) {
+        mesh.material.forEach(mat => this.disposeMaterial(mat));
+      } else {
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)